.gitignore

```diff
@@ -17,6 +17,7 @@
 */**/*dapps*
 vendor/github.com/ethereum/go-ethereum/vendor
 /node_modules/
+tags
```

Comment:
What is this?

Reply:
Tag files contain source symbols for the editor, created by `ctags`. So it's nothing for the repo.